src/args.rs

```diff
@@ -511,6 +511,9 @@ impl<'a> ArgMatches<'a> {
                     for line in io::BufReader::new(f).lines() {
                         pats.push(self.str_pattern(&line?));
                     }
+                    if Some(&self.str_pattern("")) == pats.last() {
```

Comment:
This can be replaced with `while Some(....)` to make it drop all trailing line breaks.